pkg/webui/console/views/device-overview/index.js

```diff
@@ -218,6 +218,13 @@ const DeviceInfo = ({ frequencyPlans, device, onExport }) => {
 
     if (lorawanVersion >= 100 && lorawanVersion < 110) {
       sessionInfoData.items.push(
+        {
+          key: sharedMessages.devAddr,
+          value: dev_addr,
+          type: 'byte',
+          sensitive: false,
+          enableUint32: true,
+        },
```

Comment:
This entire block can be moved above the `if` in `L219` and then you won't have to repeat it in `L238`